client/app/lib/helpers/url-helpers.js

```diff
@@ -21,5 +21,25 @@ function getUrlParameter(sParam) {
   return '';
 }
 
+/**
+ * Get the course id from URL.
+ *
+ * return {number}
+ */
+function getCourseId() {
+  const match = window.location.pathname.match(/^\/courses\/(\d+)/);
+  return match && match[1];
+}
+
+/**
+ * Get the survey id from URL.
+ *
+ * return {number}
+ */
+function getSurveyId() {
```

Comment:
I suspect there will be quite a few more functions with a similar pattern - should we abstract that? One tradeoff is that it might become less readable >_<.